backend/models/cafeModel.js

```diff
@@ -6,13 +6,36 @@ const cafeSchema = new mongoose.Schema({
     location: { type: String, required: true },
     phone: { type: String },
     website: { type: String },
+    openingHours: [
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+      { open: { type: String }, close: { type: String } },
+    ],
   },
   images: [{
     url: { type: String, required: true },
     title: { type: String, required: true },
     altText: { type: String, required: true }
   }],
-  facilities: [],
+  facilities: [String],
+
+  location: {
+    type: {
+      type: String,
+      enum: ['Point'],
+      default: 'Point',
+    },
+    coordinates: {
+      type: [Number],
+      required: true,
+    },
+  },
 });
 
+cafeSchema.index({ location: '2dsphere' });
```

Comment:
Just a q but what is this for? I get it's something needed for this location part, but interested